modal/image.py

```diff
@@ -650,6 +650,27 @@ def my_build_function():
         """
         return self.extend(build_function=(raw_function, kwargs))
 
+    def env(self, vars: dict[str, str]) -> "_Image":
+        """Sets the environmental variables of the image.
+
+        **Example**
+
+        ```python notest
+        image = (
+            modal.Image.conda()
+                .env(CONDA_OVERRIDE_CUDA="11.2")
+                .conda_install(["jax", "cuda-nvcc"], channels=["conda-forge", "nvidia"])
+                .pip_install(["dm-haiku", "optax"])
```

Comment:
These three lines need to be updated to use the right syntax.

`.env({"CONDA_OVERRIDE_CUDA": "11.2"})`

and no brackets around the list of packages for `conda_install` and `pip_install`